styles/_colors.scss

```diff
@@ -83,6 +83,7 @@ $color-tomato-08: rgba($color-tomato, 0.08);
 $color-kelley-green-48: rgba($color-kelley-green, 0.48);
 $color-kelley-green-08: rgba($color-kelley-green, 0.08);
 $color-base-50-24: rgba($color-base-50, 0.24);
+$color-base-90-40: rgba($color-base-90, 0.4);
```

Comment:
You should be updating other files so the swatch appears correctly when viewing the style-guid website. Choose another color like "color-base-50-24" and search for all uses and that should highlight the areas you need to add your new color.